protoutils/protoutils_test.go

```diff
@@ -127,6 +131,12 @@ var (
 			map[string]interface{}{"UintValue": uint(1)},
 			SingleUintStruct{},
 		},
+		{
+			"struct with errno",
+			errnoStruct,
+			map[string]interface{}{"Errno": float64(errnoVal)}, // cast float64 because pb to map conversion supports double for nums
+			ErrnoStructReturn{},
```

Comment:
Decoder fails when it needs to decode a uintptr (syscall.Errno), so we change the return type to something it can parse (int)   

```
 /Users/seanyu/Projects/goutils/protoutils/protoutils_test.go:263: Expected: nil
        Actual:   '1 error(s) decoding:
        
        * Errno: unsupported type: uintptr'
```